controllers/storagecluster/storageclasses.go

```diff
@@ -234,7 +233,9 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 // newEncryptedCephBlockPoolStorageClassConfiguration generates configuration options for an encrypted Ceph Block Pool StorageClass.
 // when user has asked for PV encryption during deployment.
 func newEncryptedCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster, serviceName string) StorageClassConfiguration {
-	encryptedStorageClassConfig := newCephBlockPoolStorageClassConfiguration(initData)
+	// PV resize of encrypted volume is not officially supported in ODF 4.10 hence setting it to False
+	allowVolumeExpansion := false
+	encryptedStorageClassConfig := newCephBlockPoolStorageClassConfiguration(initData, allowVolumeExpansion)
```

Comment:
Why not do this versus adding a new parameter that's only ever used once?

```suggestion
	// PV resize of encrypted volume is not officially supported in ODF 4.10 hence setting it to False
	allowVolumeExpansion := false
	encryptedStorageClassConfig := newCephBlockPoolStorageClassConfiguration(initData)
	encryptedStorageClassConfig.storageClass.AllowVolumeExpansion = &allowVolumeExpansion
```

Reply:
Addressed in https://github.com/red-hat-storage/ocs-operator/pull/1559